manifests/init.pp

```diff
@@ -4,8 +4,11 @@
 # menuing system with syslinux and some pxe tools installed.
 #
 class pxe (
-  $tftp_root='/srv/tftp'
```

Comment:
I'm happy to move the hardcode to params if thats easier to read, but I like being able to set `pxe::tftp_root` in heira.